src/main/java/games/strategy/engine/data/GameData.java

```diff
@@ -74,9 +74,9 @@
   private String gameName;
   private Version gameVersion;
   private int diceSides;
-  private transient List<TerritoryListener> territoryListeners = new CopyOnWriteArrayList<>();
-  private transient List<GameDataChangeListener> dataChangeListeners = new CopyOnWriteArrayList<>();
-  private transient List<GameMapListener> gameMapListeners = new CopyOnWriteArrayList<>();
+  private transient List<TerritoryListener> territoryListeners = new ArrayList<>();
+  private transient List<GameDataChangeListener> dataChangeListeners = new ArrayList<>();
+  private transient List<GameMapListener> gameMapListeners = new ArrayList<>();
```

Comment:
Can't `GameData` be accessed by multiple threads concurrently?  I don't know if listeners are ever actually added/removed/iterated concurrently, but these fields are not protected by `readWriteLock`, so there's now the potential for `ConcurrentModificationException` to be thrown while iterating.